pom.xml

```diff
@@ -103,6 +103,10 @@
         <version.semver>0.9.34-SNAPSHOT</version.semver>
         <version.springfox>2.9.2</version.springfox>
         <version.redline>1.2.9</version.redline>
+        
+        <verion.cassandra>3.11.4</verion.cassandra>
+        <verion.tinkerpop>3.4.4</verion.tinkerpop>
```

Comment:
Both properies have have typo -- it's `verion` instead of `version`.

Reply:
fixed